my_2d_robot/src/my_2d_robo_state_publisher.cpp

```diff
@@ -0,0 +1,47 @@
+#include <string>
```

Comment:
Run a clang formatter on your code that conforms to the ros styleguide: https://github.com/davetcoleman/roscpp_code_format